### DISCUSSION TITLE
Can't get italics to work

### DISCUSSION THREAD

**Discussion by twiclo** [kovidgoyal/kitty#7873] [Q&A] [answered]
With my default font settings of 'monospace' and 'auto' for everything I am unable to print an italic word to my console. When I try to specify NotoSans I can see kitty picking it up for all the styles except italic
```
font_family      'NotoSansM Mono'
bold_font        'NotoSansM Mono Bold'
italic_font      'NotoSansM Mono Italic'
bold_italic_font 'NotoSansM Mono Bold Italic'
```
```
[tanner@sato ~]$ kitty --debug-font-fallback
[0.171] Text fonts:
[0.171]   Normal: NotoSansMono-Regular: /usr/share/fonts/google-noto/NotoSansMono-Regular.ttf:0
Features: ()
[0.171]   Bold: NotoSansMono-Bold: /usr/share/fonts/google-noto/NotoSansMono-Bold.ttf:0
Features: ()
[0.171]   Italic: NotoSansMono-Regular: /usr/share/fonts/google-noto/NotoSansMono-Regular.ttf:0
Features: ()
[0.171]   Bold-Italic: NotoSansMono-Bold: /usr/share/fonts/google-noto/NotoSansMono-Bold.ttf:0
Features: ()
```

I'm not sure what I need to set for my values. There is no italic font in `/usr/share/fonts/google-noto` but I do have them in `.local/share/fonts`. I even tried pointing to them directly
```
italic_font      '/home/tanner/.local/share/fonts/NotoSansNerdFont-Italic.ttf'
```
I know the FAQ says to not use nerd fonts but I figured I should at least be able to see an italic font with this method. I can switch to a non nerd font afterwards.
I don't care too much to have Noto being used. I'm surprised that the default font doesn't have italic support. Looking for suggestions

**Comment by twiclo** [comment]
Okay I got it to work using `list-fonts` and went with Source Code Pro. I would still like to know what is the correct way to use Noto, however.

**Reply by kovidgoyal** [reply]
https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

**Reply by twiclo** [reply, marked as answer]
The answer from that link is that kitty only supports monospace fonts. There is no monospaced italic version of Noto
